src/Nncase.Core/Utilities/QuantUtility.cs

```diff
@@ -149,4 +149,320 @@ public static List<float> GetWeightsRangesByChannel(Span<float> weights, int cha
 
         return minMaxArr;
     }
+
```

Comment:
Move QuantUtility.cs to Nncase.Quantization instead of Nncase.Core. Otherwise all other modules will dependent on ortki.